apps/docs/content/getting-started/animation.mdx

```diff
@@ -0,0 +1,247 @@
+---
+title: Animation
+storybook: '/brand/storybook/?path=/story/components-animations--playground'
+description: Use animations to add visual interest and interactivity to a web page or application.
+---
+
+import {PropTableValues} from '../../src/components'
+import {Link} from 'gatsby'
+import {AnimationVariants, Label} from '@primer/react-brand'
+import {supportedComponents} from '../../scripts/components-with-animation'
+
+Certain Primer Brand components include animation presets. These components are:
+
+<PropTableValues
+  values={supportedComponents}
+  commaSeparated
+  removeApostrophes
+/>
+
+<br />
+<br />
+
+You can use the `animate` prop available on these components to apply one of the available animation presets directly to the component.
+
+<strong>Example:</strong>
```

Comment:
Do we need this? I think the live example box is clear when we use it as "example"
```suggestion
```